mainapp/models.py

```diff
@@ -53,6 +53,8 @@ class Request(models.Model):
     location = models.CharField(max_length=500,verbose_name='Location - സ്ഥലം')
     requestee = models.CharField(max_length=100,verbose_name='Requestee - അപേക്ഷകന്‍റെ പേര്')
     requestee_phone = models.CharField(max_length=10,verbose_name='Requestee Phone - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍')
+    requestee_phone_2 = models.CharField(max_length=10,verbose_name='Requestee Phone (2) - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍ (2)',null=True,)
```

Comment:
For me it seems `blank=True` is good (or not needed if `null=True` act as not required while form validation). As that field can skip while filling

Reply:
Updated the repo. Migrations added. Field check added when displaying phone number.

Reply:
Exactly. i was going to comment here that. Thank you @latheefts

Reply:
For CharField, don't put null=True, just use blank=True